src/components/BaseLayerToggler/BaseLayerToggler.js

```diff
@@ -24,6 +24,16 @@ const propTypes = {
    */
   className: PropTypes.string,
 
+  /**
+   * Children of the previous baselayer button.
+   */
+  previousLayerBtChildren: PropTypes.node,
```

Comment:
How about naming these props `prevButtonContent` and `nextButtonContent`? This would match the `titles.prevButton` prop naming in line 52, 53 and does not leak too much implementation details.

Reply:
Yes good proposition thanks!